notebooks/amazon_forest_notebook.py

```diff
@@ -208,7 +209,7 @@
 
 # <codecell>
 
-from keras.callbacks import ModelCheckpoint
+from tensorflow.contrib.keras.api.keras.callbacks import ModelCheckpoint
 
 filepath="weights.best.hdf5"
 checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True, mode='max')
```

Comment:
Are you sure we want to keep the `verbose=1`? It really generates ugly output and doesn't give us much useful information:
<img width="1005" alt="amazon_forest_notebook" src="https://cloud.githubusercontent.com/assets/4030626/26435667/de93a918-40de-11e7-8f02-dcf3c7d9d83f.png">

Something like this would be best:
`checkpoint = ModelCheckpoint(filepath, monitor='val_acc', save_best_only=True)`
Wdyt?